chaoslib/pumba/pod_failure_by_sigkill.yml

```diff
@@ -145,18 +146,18 @@
             kubectl delete -f /chaoslib/pumba/pumba_kube.yml -n {{ namespace }}
           args:
             executable: /bin/bash
-          when: pumba_job_result.rc == 0
+          when: pumba_pod_result.rc == 0
         
-        - name: Confirm that the pumba job is not present
+        - name: Confirm that the pumba pod is not present
           shell: >
-            kubectl get pods -l job-name=pumba-sig-kill-{{ run_id }} --no-headers -n {{ namespace }}
+            kubectl get pods -l name=pumba-sig-kill-{{ run_id }} --no-headers -n {{ namespace }}
           args:
             executable: /bin/bash
           register: result
           until: "'No resources found' in result.stderr"
           delay: 2
-          retries: 60
-      when: "pumba_job_result is defined"
+          retries: 90
+      when: "(pod_run_status is defined and pod_run_status == 'Running') and pumba_pod_result is defined"
```

Comment:
Is this mandatory?

Reply:
Ok it is because of invalid state of pod?

Reply:
Yes, If the pod_run_status is not `Running` then the util itself deleted the pumba pod.

Reply:
cool